TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java

```diff
@@ -141,7 +142,12 @@ public SampleMecanumDrive(HardwareMap hardwareMap) {
         setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
 
         if (RUN_USING_ENCODER && MOTOR_VELO_PID != null) {
-            setPIDCoefficients(DcMotor.RunMode.RUN_USING_ENCODER, MOTOR_VELO_PID);
+            Log.i("voltage", Double.toString(hardwareMap.voltageSensor.iterator().next().getVoltage()));
```

Comment:
The quickstart doesn't have much logging of this kind, but I suppose it's fine. I do prefer `RobotLog` over `Log`.

Reply:
This was an error on my part. Accidentally committed the lines I was using for testing. I was going to remove it but decided to just remove it in the same commit that I would be making any changes to the default voltage to divide by.